index.js

```diff
@@ -195,12 +228,12 @@ client.on("messageCreate", async (message) => {
                 return response.author.id === message.author.id;
             };
 
-            const collector = new MessageCollector(message.channel, { time: 60000, max: 1, filter: filter });
+            const collector = new MessageCollector(message.channel, { time: SIXTY_THOUSAND, max: 1, filter: filter });
 
             // Event listener for receiving title:
             collector.on("collect", async (response) => {
-                talkingPointCount++;
-                const title = `TP${String(talkingPointCount).padStart(2, "0")} - ${response.content}`;
+                talkingPointCounter++;
```

Comment:
I prefer doing `talkingPointCounter += 1` but that's up to preference. I always do it because it's more clear than `++` and there can be some strange bugs with `++` and it can be more easily updated to be `+= 2` if needed